python_code_style.md

```diff
@@ -0,0 +1,119 @@
+# Python Code Style
+
+We use [pylint](https://www.pylint.org/) to perform automatic code checks on our
+repositories. It helps enforce code standard, and also does some static code
+analysis for potential programming errors.
+
+## Setting up a new project
+
+When creating a new repository that contains Python code, please add into the
+repository root folder a file named `.pylintrc` with the following contents:
```

Comment:
How about just putting a `.pylintrc` file into this repository, that people can easily copy? Maybe just provide the command to copy the file from the GitHub blob.